### ISSUE TITLE
"Zusatzmaterialien"

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#228]
Decide how to integrate
*Bühnenzeichnungen
*Soufflierbücher

**Comment by gerritbruening** [comment]
https://faustedition.uni-wuerzburg.de/wiki/index.php/30.%E2%80%9331._M%C3%A4rz_2017#Zusatzmaterialien_2

**Comment by thvitt** [comment]
Sind die Bühnenzeichnungen und Soufflierbücher einfach neue Zeugen oder wie liegen die vor und wie sollen die dargestellt werden?

**Comment by gerritbruening** [comment]
> Bühnenzeichnungen und Soufflierbücher

Ja, es sind quasi Zeugen, aber ohne Transkriptionen und ohne Metadaten.

**Comment by thvitt** [comment]
Gibt's 'ne Lagenstruktur? (Wobei: Die gibt's bei den Drucken auch nicht, oder?)

Bräuchten dann:

* angepasste Version des DocumentViewers
* Sigle
* (minimale) Metadatendatei mit:
* Liste der Seiten oder Lagenstruktur

**Comment by gerritbruening** [comment]
Nein, keine Lagenstruktur oder irgendetwas. Nur die Info, worum es sich grundsätzlich handelt.

**Comment by gerritbruening** [comment]
Telko 29.9.2018:
Button soll **Materialien** heißen, darunter sollen wiederum als Button erscheinen:
-	„Schillemeit, Vorarbeiten zu einer Geschichte des Faust“
-	Wasserzeichen
-	Hyperspektralaufnahmen
-	RFA-Bericht
Und einer noch ferneren Zukunft:
* Theaterzeichnungen
* Soufflierbücher

**Comment by gerritbruening** [comment]
Habe das [oben gesagte](https://github.com/faustedition/faust-web/issues/228#issuecomment-426226306) umgesetzt.
Es fehlen noch die eigentlichen Archivmatierialien, außerdem Theaterzeichnungen und Soufflierbücher. Wollen wir für die letzteren auch schon Buttons einrichen und WIP-Seiten oder so einrichten?

**Comment by gerritbruening** [comment]
Zum RFA-Bericht kann ich am Montag die pdf besorgen, ich weiß aber nicht, ob da so ohne weiteres ein Download eingerichtet werden kann?

**Comment by gerritbruening** [comment]
Ist mir alles recht: Link auf Github-Ordner, besser passender Seitentitel.

**Comment by thvitt** [comment]
nee, link direkt aufs file.
